libs/logic-apps-shared/src/designer-client-services/lib/uiInteractionsService.ts

```diff
@@ -0,0 +1,21 @@
+import type { UiInteractionData } from '../../utils/src/lib/models/uiInteractionData';
+import { AssertionErrorCode, AssertionException } from '../../utils/src';
+import type { TopLevelDropdownMenuItem } from '../../utils/src/lib/models/topLevelDropdownMenuItem';
+
+
+export interface IDesignerUiInteractionsService {
+    getAddButtonDropdownMenuItems?: (previousNodeData: UiInteractionData | undefined) => TopLevelDropdownMenuItem[];
```

Comment:
```suggestion
    getAddButtonMenuItems?: (previousNodeData: UiInteractionData | undefined) => TopLevelDropdownMenuItem[];
```

Per the discussion thread